vidur/utils/mfu_calculator.py

```diff
@@ -1,21 +1,25 @@
-from vidur.config import Config
+from vidur.config import SimulationConfig
 from vidur.entities import BatchStage
 from vidur.utils.param_counter import ParamCounter
 
 
 class MFUCalculator:
-    def __init__(self, config: Config):
+    def __init__(self, config: SimulationConfig):
```

Comment:
nit: we should pass the minimum possible subset of the config to any class, it helps a lot when refactoring

Reply:
agreed, the narrowest possible interface.